### DISCUSSION TITLE
Unable to login with Admin User

### DISCUSSION THREAD

**Discussion by sumitj12** [strapi/strapi#7685] [Legacy Topics]
Hello Everyone, 

Can someone please help me with login as a admin user in strapi CMS? Initially i ran the command 
**yarn create strapi-app my-project --quickstart** for initial setup with account registration. 

Next day when i tried to login again with same user, i am unable to find from where i can login again. The user have its own content data configured in strapi.  
please help me understand this?

Thank you.

**Comment by derrickmehaffy** [comment]
We need more information as to the error you see
